README.md

```diff
@@ -1,3 +1,41 @@
-#Hollow API Generator Nebula Plugin
+=======
+# Hollow API Generator Nebula Plugin
 
-This plugin will provide a task for generating a Hollow consumer API, as described [here](http://hollow.how/getting-started/#consumer-api-generation).
+This plugin provides task for generating hollow consumer api, that is 
+described [here](http://hollow.how/getting-started/#consumer-api-generation)
+
+## Usage
+In order to use, add plugin dependency to your buildscript
+```
+buildscript {
+    repositories {
+        mavenLocal()
+        jcenter()
+    }
+    dependencies {
+        classpath "com.netflix.nebula:nebula-hollow-plugin:0.1"
+    }
+}
+```
+apply it
+```
+apply plugin: 'nebula.hollow-plugin'
```

Comment:
We want this to be `apply plugin: 'nebula.hollow'`. I wonder what we need to change to make it work that way?